src/Core/Services/MetadataProviders/MetadataProviderFactory.cs

```diff
@@ -88,5 +89,19 @@ public IEnumerable<ISqlMetadataProvider> ListMetadataProviders()
         {
             return _metadataProviders.Values;
         }
+
+        /// <inheritdoc />
+        public void InitializeAsync(
```

Comment:
Is it possible to add a test case for this to ensure that the correct metadataprovider is getting picked up by this function.

Reply:
created a work item : "https://dev.azure.com/powerbi/AppDev/_backlogs/backlog/AppDev%20Team/Epics?workitem=1356906" as discussed offline

Reply:
as discussed offline